dali/pipeline/operators/fused/crop_cast_permut_test.cc

```diff
@@ -39,7 +39,7 @@ TYPED_TEST(CropCastPermuteTest, Layout_DALI_NHWC) {
 }
 
 TYPED_TEST(CropCastPermuteTest, Layout_DALI_SAME) {
-  const OpArg params[] = {{"crop", "224, 224", DALI_FLOAT_VEC}, {"output_layout", "3", DALI_INT32}};
+  const OpArg params[] = {{"crop", "224, 224", DALI_FLOAT_VEC}, {"output_layout", "4", DALI_INT32}};
```

Comment:
this "4" (or "3") is extremely ugly. Can we do anything better?

Reply:
Replaced by something ugly, but less ugly :D